jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TabsUnitTests.java

```diff
@@ -0,0 +1,80 @@
+package io.github.epam.angular.tests.unit;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.tabsSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static org.testng.Assert.assertTrue;
+
+public class TabsUnitTests extends TestsInit {
+
+    private final List<String> TABS_TITLES_DEFAULT_LIST = Arrays.asList("First", "Second", "Third");
+    List<String> TABS_LINKS_TITLES = Arrays.asList("First", "Second", "Third", "Disabled Link");
+
+    @BeforeMethod
```

Comment:
Are you sure that @BeforeMethod will run before all methods even if the previous test failed?
In my opinion it is necessary use attribute "alwaysRun" with value "true".